diktat-rules/src/test/resources/test/paragraph4/null_checks/IfConditionBreakCheckTest.kt

```diff
@@ -0,0 +1,85 @@
+package test.paragraph4.null_checks
+
+
+fun foo() {
+    var result: Int? = 19
+    while(result != 0 ) {
+        if (result == null) {
+            foo()
+        } else {
+            break
+        }
+    }
+}
+
+fun foo() {
+    var result: Int? = 19
+    while(result != 0 ) {
+        if (result == null) {
+            break
+        } else {
+            foo()
+        }
+    }
+}
+
+fun foo() {
+    var result: Int? = 19
+    while(result != 0 ) {
+        if (result != null) {
+            break
+        } else {
+            foo()
+        }
+    }
+}
+
+fun foo() {
+    var result: Int? = 19
+    while(result != 0 ) {
+        if (result != null) {
+            foo()
+        } else {
+            break
+        }
+    }
+}
+
+////////////////////////////////
+
+fun foo() {
```

Comment:
I believe such test (without `break`) should be in regular tests for this rule, no need to double-check it here